yarn.lock

```diff
@@ -422,6 +422,11 @@ cross-fetch@3.1.5:
   dependencies:
     node-fetch "2.6.7"
 
+crypto-js@4.2.0:
+  version "4.2.0"
```

Comment:
It looks like the sha1.js has version CryptoJS v3.0.2, so is this also an upgrade? And have you looked into what this upgrade entails?

Reply:
Excellent point, I hadn't noticed. I went to the [release notes](https://www.npmjs.com/package/crypto-js#release-notes) and think it's fine to do the upgrade. The breaking change in 4.0 is for environments that don't have the native crypto module. It's well-supported at this point ([docs](https://developer.mozilla.org/en-US/docs/Web/API/Crypto)).